spring-kafka-docs/src/main/antora/modules/ROOT/pages/retrytopic/how-the-pattern-works.adoc

```diff
@@ -13,7 +13,21 @@ IMPORTANT: By using this strategy you lose Kafka's ordering guarantees for that
 
 IMPORTANT: You can set the `AckMode` mode you prefer, but `RECORD` is suggested.
 
-IMPORTANT: At this time this functionality doesn't support class level `@KafkaListener` annotations.
+IMPORTANT: This functionality now supports class level `@KafkaListener` annotations since version 3.2.
```

Comment:
Thank you for looking into this!
However  I believe there is not need in this section here at all.
It is really mentioned and explained in the next chapter just after this intro: https://docs.spring.io/spring-kafka/reference/retrytopic/retry-config.html#using-the-retryabletopic-annotation.

So, please, just remove this `IMPORTANT` and that's it.
You can wrap that `@KafkaListener` in the next chapter into a code snippet though 😄

Reply:
Ah, I see, it's clearly explained in the next chapter, but I wrote it redundantly 😅
Thank you for the quick feedback!